### PR TITLE
Add support for country rank

### PR DESCRIPTION
**Pull request by tgi74**
the country rank was always zero (it was not parsed by json)

### CODE REVIEW

**Comment by idSoftwareWolf** [comment]
The osu discord is checking this right now! :)

**Review by peppy — DISMISSED** [review]

File: osu.Game/Overlays/Profile/RankGraph.cs [hunk 1]
```diff
@@ -124,9 +124,9 @@ private void userChanged(User user)
 
         private void updateRankTexts()
         {
-            rankText.Text = User.Value.Statistics.Rank > 0 ? $"#{User.Value.Statistics.Rank:#,0}" : "no rank";
+            rankText.Text = User.Value.Statistics.Ranks.Global > 0 ? $"#{User.Value.Statistics.Ranks.Global:#,0}" : "no rank";
             performanceText.Text = User.Value.Statistics.PP != null ? $"{User.Value.Statistics.PP:#,0}pp" : string.Empty;
-            relativeText.Text = $"{User.Value.Country?.FullName} #{User.Value.CountryRank:#,0}";
+            relativeText.Text = $"{User.Value.Country?.FullName} #{User.Value.Statistics.Ranks.Country:#,0}";
```
Comment (Aergwyn) [inline osu.Game/Overlays/Profile/RankGraph.cs]: Can country rank also be zero? I'd assume that could be the case if there's a check for the global one displaying "~~unranked~~ no rank" instead.

Comment (peppy) [reply osu.Game/Overlays/Profile/RankGraph.cs]: nice catch

**Comment by Aergwyn** [comment]
May I suggest labelling it *unranked* instead of *no rank*?

**Review by peppy — APPROVED** [review]

**Pull request merged**